distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -65,3 +65,4 @@ export JAVA_MAX_MEM=4096M
 # export KARAF_BASE # Karaf base folder
 
 export KARAF_OPTS="-Dfile.encoding=UTF8 -Dddf.home=$DDF_HOME"
+export JAVA_OPTS=-"server -Xms128M -Xmx4096M -XX:+UnlockDiagnosticVMOptions -XX:+UnsyncloadClass"
```

Comment:
✏️ I don't think we should be doing this; this will ignore any settings provided by the user in their environment. We could simply tack on to the `EXTRA_JAVA_OPTS` variable